Engine/DataFeeds/FileSystemDataFeed.cs

```diff
@@ -502,6 +522,10 @@ private IEnumerator<BaseData> ConfigureEnumerator(SubscriptionRequest request, b
             // optionally apply fill forward logic, but never for tick data
             if (request.Configuration.FillDataForward && request.Configuration.Resolution != Resolution.Tick)
             {
+                var subscriptionConfigs = _subscriptions.Select(x => x.Configuration).Concat(new[] { request.Configuration });
+
+                UpdateFillForwardResolution(subscriptionConfigs);
```

Comment:
In this case we become dependent on the logic of the UpdateFillForwardResolution() itself. I would like fillforward resolution update logic to be encapsulated in the method. If one wants to change it later, this change should not affect the lines above.